File: c++/ARRAY/prime.cpp
#include<iostream>
using namespace std;
int main()
{
    bool isPrime=1;
    int num;
    cout<<"Enter the number:";
    cin>>num;
    for(int i=2; i<num; i++)
    {
        if(num%i==0)
        {
            isPrime=0;
            break;
        }
    }
        if(isPrime==0)
        {
            cout<<"Not a Prime number";
        }
        else
        {
            cout<<"Prime number";
        }
}

File: java/array/Prime.java
package array;
public class Prime {
    static boolean prime(int num) {
        // boolean isPrime;
        for (int i = 2; i <= num; i++) {
            if (num % i == 0)
                return false;
            break;
        }
        return true;
    }

    public static void main(String[] args) {
        int num = 583;
        boolean res = prime(num);
        if (res == false)
            System.out.println("Not a prime");
        else
            System.out.println("Prime number");
    }
}
